docs/sources/flow/reference/components/otelcol.exporter.loadbalancing.md

```diff
@@ -0,0 +1,263 @@
+---
+title: otelcol.exporter.loadbalancing
+labels:
+  stage: beta
+---
+
+# otelcol.exporter.loadbalancing
+
+{{< docs/shared lookup="flow/stability/beta.md" source="agent" >}}
+
+`otelcol.exporter.loadbalancing` accepts logs and/or traces from other `otelcol` components
+and writes them over the network using the OpenTelemetry Protocol (OTLP) protocol. 
+
+> **NOTE**: `otelcol.exporter.loadbalancing` is a wrapper over the upstream
+> OpenTelemetry Collector `loadbalancing` exporter. Bug reports or feature requests will
+> be redirected to the upstream repository, if necessary.
+
+Multiple `otelcol.exporter.loadbalancing` components can be specified by giving them
+different labels.
+
+Note that either a trace ID or a service name is used for the decision on which backend to use. 
+The actual backend load isn't taken into consideration. Even though this load-balancer won't do 
+round-robin balancing of the batches, the load distribution should be very similar among backends 
+with a standard deviation under 5% at the current configuration.
+
+`otelcol.exporter.loadbalancing` is especially useful for backends configured with tail-based samplers
+which make a decision based on the view of the full trace.
+
+When a list of backends is updated, around `1/n` of the space will be changed, so that the same trace ID 
+might be directed to a different backend, where `n` is the number of backends. This should be stable enough 
+for most cases, and the higher the number of backends, the less disruption it should cause.
```

Comment:
The otel [docs](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.63.0/exporter/loadbalancingexporter) also mention this:

> Still, if routing stability is important for your use case and your list of backends are constantly changing, consider using the groupbytrace processor. This way, traces are dispatched atomically to this exporter, and the same decision about the backend is made for the trace as a whole.

However I don't want to include this, because we don't have a groupbytrace processor and it'd be confusing to mention it imo.